README.md

```diff
@@ -19,7 +19,7 @@ For more info, see:
 ## Development
 
 1. `npm install`
-2. `./node_modules/.bin/web-ext run -s src/`
+2. `./node_modules/web-ext/bin/web-ext run -s src/`
```

Comment:
Could be written as `npx web-ext run -s src` as well, or moved into a package script. Directly linking to the script inside the module could fail in the future, since its location might change.